Explicitly specify which categories of dtypes the elementwise functions are defined for (#121)
data-apis/array-api: spec/API_specification/elementwise_functions.md

-   Optional parameters must be [keyword-only](https://www.python.org/dev/peps/pep-3102/) arguments.
-   Broadcasting semantics must follow the semantics defined in {ref}`broadcasting`.
-   Unless stated otherwise, functions must support the data types defined in {ref}`data-types`.

> -   Functions may only be required for a subset of input data type. Libraries may chose to implement functions for additional data types, but that behavior is not required by the spec. See {ref}`data-type-categories`.

Reviewer: ```suggestion
-   Functions may only be required for a subset of input data type. Libraries may choose to implement functions for additional data types, but that behavior is not required by the specification. See {ref}`data-type-categories`.
```

Not entirely clear to me what is meant by the following sentence:

> Functions may only be required for a subset of input data type

-   **x1**: _&lt;array&gt;_

>     -   input array corresponding to the y-coordinates. Should have a floating-point data type

Reviewer: ```suggestion
    -   input array corresponding to the y-coordinates. Should have a floating-point data type.
```

-   **x**: _&lt;array&gt;_

    -   input array. Should have a numeric data type.

#### Returns

-   **out**: _&lt;array&gt;_

>     -   an array containing test results. An element `out_i` is `True` if `x_i` is `NaN` and `False` otherwise. The returned array Should have a data type of `bool`.

Reviewer: ```suggestion
    -   an array containing test results. An element `out_i` is `True` if `x_i` is `NaN` and `False` otherwise. The returned array should have a data type of `bool`.
```

-   **x1**: _&lt;array&gt;_

    -   first input array. Should have a numeric data type.

-   **x2**: _&lt;array&gt;_

>     -   second input array. Must be compatible with `x1` (see {ref}`broadcasting`). Should have a numeri

Reviewer: Missing rest of sentence.
